dev/components/conversationsList/tpl.pug

```diff
@@ -6,20 +6,20 @@
       .main-content
         .content
           .message-info.ellipsis
-            span(ng-if="!vm.toUser && item.email && item.email != 'null' && item.email != 'undefined'") to: {{item.email}}
-            span(ng-if="!vm.toUser && !item.email || item.email === 'null' || item.email === 'undefined'") Public
+            span(ng-if="!vm.toUser && item.email && item.email != null && item.email != undefined") to: {{item.email}}
```

Comment:
If "item.email" will be null or undefined if will not pass this part "!vm.toUser && item.email" so looks like " && item.email != null && item.email != undefined" will never run